### DISCUSSION TITLE
Decimal rounding - logged metrics

### DISCUSSION THREAD

**Discussion by Arderiu** [Lightning-AI/pytorch-lightning#19496] [Lightning Trainer API: Trainer, LightningModule, LightningDataModule]
Dear community,

In the training_step function of my LightningModule, I am logging the train and validation loss:
`self.log('train_loss', loss, on_step=False, on_epoch=True, prog_bar=True)`

However, the logged metrics are rounded to the nearest integer by default. Do you know how can I control the number of decimals shown? Thank you in advance!
